README.md

```diff
@@ -1,6 +1,10 @@
-# Code PushUp CLI
+<h1 align="center">Code PushUp CLI</h1>
+<img alt="Code-Pushup Logo" src="./packages/cli/docs/images/logo.png" width="985">
```

Comment:
:art: I think stretching the logo to the full width is overkill, looks better with a smaller width and centered.

```suggestion
<p align="center"><img alt="Code-Pushup Logo" src="./packages/cli/docs/images/logo.png" width="500" /></p>
```

<details><summary> :framed_picture: before </summary>
<p>

![image](https://github.com/code-pushup/cli/assets/34691111/4164d604-8e14-4bd9-9819-c70f2cda6df6)


</p>
</details> 

<details><summary> :framed_picture: after </summary>
<p>

![image](https://github.com/code-pushup/cli/assets/34691111/af844d18-f356-429e-a621-8c7b11f41296)


</p>
</details>